Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2494,7 +2494,11 @@ extension BrowserViewController: ContextMenuHelperDelegate {
             }
             
             let openInNewTabAction = UIAlertAction(title: Strings.OpenImageInNewTabActionTitle, style: .default) { _ in
-                self.tabManager.addTab(URLRequest(url: url), afterTab: self.tabManager.selectedTab)
+                if PrivateBrowsingManager.shared.isPrivateBrowsing {
+                    self.tabManager.addTabAndSelect(URLRequest(url: url), afterTab: self.tabManager.selectedTab, isPrivate: true)
```

Comment:
Should probably use the same call for both states. Currently this uses `addTabAndSelect` for private and just `addTab` for normal.

Probably best to just use the original call and add `isPrivate` to it like: 

```swift
let isPrivate = PrivateBrowsingManager.shared.isPrivateBrowsing
self.tabManager.addTab(URLRequest(url: url), afterTab: self.tabManager.selectedTab, isPrivate: isPrivate)
```